plugins/uefi-capsule/fu-plugin-uefi-capsule.c

```diff
@@ -402,6 +446,9 @@ fu_plugin_update (FuPlugin *plugin,
 	str = _("Installing firmware update…");
 	g_assert (str != NULL);
 
+	if (fu_capsule_update_on_disk(device, error))
+			return TRUE;
+
```

Comment:
I think you should detect during coldplug if capsules on disk is supported, and then install separately.  It shouldn't be a runtime decision, OsIndications will be set before OS loads.

Reply:
That would be OsIndicationsSupported, but yea I think I get the idea